<launch>

  <node name="camera_driver" pkg="usb_cam" type="usb_cam_node" args="_video_device:=/dev/video1 _pixel_format:=yuyv" />

  <node name="facial_recognition" pkg="me495_vision" type="facial_recognition.py" output="screen"/>

  <!-- Limb Argument -->
  <arg name="limb" default="right" />

  <!-- Start the Joint Trajectory Action Server -->
  <node name="sdk_joint_trajectory_action_server" pkg="intera_interface" type="joint_trajectory_action_server.py" args="--limb $(arg limb) --mode position_w_id" required="true" />

  <!-- Run the Joint Trajectory Action Test Example -->
  <node name="drawing" pkg="me495_vision" type="drawing.py" args="--limb $(arg limb)" output="screen" />

</launch>
